pywemo/ouimeaux_device/dimmer.py

```diff
@@ -1,37 +1,13 @@
-from . import Device
+from .switch import Switch
 
-
-class Dimmer(Device):
-
-    def set_state(self, state):
-        """
-        Set the state of this device to on or off.
-        """
-        self.basicevent.SetBinaryState(BinaryState=int(state))
-        self._state = int(state)
-
-    def off(self):
-        """
-        Turn this device off. If already off, will return "Error".
-        """
-        return self.set_state(0)
-
-    def on(self):
-        """
-        Turn this device on. If already on, will return "Error".
-        """
-        return self.set_state(1)
-
-    def toggle(self):
-        """
-        Toggle the dimmer's state.
-        """
-        return self.set_state(not self.get_state())
-		
+class Dimmer(Switch):		
     def set_brightness(self, brightness):
```

Comment:
Might also be worth adding a `get_brightness()` call - I imagine you'll want this for HA. 

You will want to get this from the device so that you have the right value even if it wasn't set via Python.

Also worth checking that if you turn the device on / off via the wemo app and via the self.on() / self.off() - the `get_brightness` call always returns the right number.

Might need to add some explicit setting of `self._brightness`.

Reply:
How would I pull the value from the device? 

>>> device.basicevent.GetBinaryState()
{'fader': '300:-1:1:0:100', 'BinaryState': '1', 'brightness': '75'}

Reply:
`device.basicevent.GetBinaryState().get('brightness')`

Reply:
Plus you may want to make it a number 

This is from elsewhere in the code base
````
            try:
                self._state = int(state.get('BinaryState', 0))
            except ValueError:
                self._state = 0
````

Reply:
I tried and tried to figure your method out, and couldn't get it :( hope this way is acceptable!